doc/AR_RPC_Layer.md

```diff
@@ -37,7 +37,7 @@ Bulk in the rpc layer deals with the network part of a complete 0-copy path. The
 
 Bulk implementation interacts closely with the rdma capabilities exported by the underlying transport. At the moment, Motr uses ONC RPC over TCP as a transport. ONC RPC, which by itself is stream-based, has an rdma extension. This extension is described in rfc5666. rfc5666 introduces rdma at the level of XDR encoding, making it transparent for the higher layers. Motr has an (almost completed) support for rfc5666 in the "rdma" branch. On the client side, this implementation uses rfc5666 compliant sunrpc module in Linux kernel, and on the server side, it uses Infiniband verbs through a user space ibverbs library.
 
-Upper layers, including rpc layer, access network transport through interfaces defined in core/net/. These interfaces might require changes to accommodate for the new rpc layer features. Specifically, an interface for rdma must be added that would be compatible with the LNET. For the time being (until LNET support is added) this interface should be backed by a dummy implementation.  
+Upper layers, including rpc layer, access network transport through interfaces defined in core/net/. These interfaces might require changes to accommodate for the new rpc layer features. Specifically, an interface for rdma must be added that would be compatible with the LNET. For the time being (until LNET support is added) this interface should be backed by a stub implementation.  

```

Comment:
Please move this entire document to an "outdated" directory. Edits, like the replacement of "Mero" with "Motr" only add confusion: Motr never ever used "ONC RPC over TCP" as a transport.